pkg/agent/ngt/handler/grpc/handler.go

```diff
@@ -50,6 +51,8 @@ func New(opts ...Option) Server {
 }
 
 func (s *server) Exists(ctx context.Context, uid *payload.Object_ID) (res *payload.Object_ID, err error) {
+	ctx, span := trace.StartSpan(ctx, "vald/agent-ngt.Exists")
```

Comment:
if observe disable we shouldn't use trace package

Reply:
That's right. :+1:

Do you have any ideas to accomplish it?
I don't want to use 'if' statements in every handlers... :thinking:

Reply:
Okay.
I'll wrap trace.StartSpan function like the followings:
```go
func SpartSpan(ctx context.Context, name string, o ...trace.StartOption) (context.Context, *Span) {
  if !enabled {
    return ctx, nil
  }
return trace.StartSpan(ctx, name, o)
}
```